### ISSUE TITLE
Series with no 'Airdate' show as 0% complete

### ISSUE THREAD

**Issue by extrobe** [SickChill/SickChill#4329]
If you have all episodes of a TV series, but none of the episodes have an 'Aired Date', the entire series is listed as being 0/0 , and marked as 0% complete

Example: [Zambezi (1965)](https://www.thetvdb.com/?tab=season&seriesid=287007&seasonid=600099&lid=7)

The 0/0 makes sense, given (as far as SR is concerned) it never aired, but would be good if they were not marked as being _incomplete_.

~

Sickrage running on Master branch, within unRaid docker
- won't include logs as don't imagine they'll tell you much, but please let me know if that's incorrect!

**Comment by rutskiuk** [comment]
I have noticed SR marking a series as incomplete if a show does not have airdates.

The first thing I would do wherever possible is update theTVDB.com entry with the airdates. Sometimes this is difficult but the example show provided was really easy so I've updated them now and that show will show 3/3 and complete after an update.

My situation is that I have some shows that weren't aired on TV

[](url)www.thetvdb.com/?tab=series&id=250998

I can't add airdates to that (direct to dvd) so have the same problem with the show listed as incomplete even though I have each of them.
